genai-on-vertex-ai/natural_language_to_sql/natural_language_to_sql.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>Instead of asking user to uncomment/comment, consider adding this block to make it work on both Colab and Vertex AI Workbench.</p><p><code></code><code></p><p>import sys</p><p> </p><p>if "google.colab" in sys.modules:</p><p>  ...</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>